DTXCreatorProject/Code/00.App/CMainForm.Designer.cs

```diff
@@ -423,7 +423,7 @@ private void InitializeComponent()
 			// hScrollBarBLEVEL
 			// 
 			resources.ApplyResources( this.hScrollBarBLEVEL, "hScrollBarBLEVEL" );
-			this.hScrollBarBLEVEL.Maximum = 109;
+			this.hScrollBarBLEVEL.Maximum = 999 + 9;
```

Comment:
For some reason all maximum values need to be +9. It's a result of a bug somewhere but I have no idea why.